### PR TITLE
Updated the youtube section

### LINKED ISSUE #1229: Data Structures playlist with c++
### Site URL

https://www.youtube.com/playlist?list=PLDzeHZWIZsTryvtXdMr6rPh4IDexB5NIA

### Description of the above provided site

Complete C++ Placement DSA Course, I followed for preparing Data Structures, covers all topic in depth and has questions and tasks as well, you can also track other students performance.

### Category

Youtube

### Sub Category

Data Structures

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by anshulsoni4**
## fixes Data Structures playlist with c++ #1229

Closes #1229 

## Changes proposed

Added the DSA playlist  using the c++ language

### CODE REVIEW

**Comment by k-deepak04** [comment]
@anshulsoni4 please resolve the conflicts, rebase your branch with the main one.

**Review by CBID2 — APPROVED** [review]
Hi @anshulsoni4. I made a few tweaks, but other than that, you did a good job! :) Now, you just have to wait for another review before your PR can be merged! :)

**Comment by anshulsoni4** [comment]
> Hi @anshulsoni4. I made a few tweaks, but other than that, you did a good job! :) Now, you just have to wait for another review before your PR can be merged! :)

thanks man!

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @anshulsoni4! :) There are some merge conflicts in your branch so I highly recommend solving them.

**Comment by anshulsoni4** [comment]
> Hi @anshulsoni4! :) There are some merge conflicts in your branch so I highly recommend solving them.

pls check now

**Review by CBID2 — APPROVED** [review]
LGTM

File: database/youtube/data-structures.json
Comment (rupali-codes) [inline database/youtube/data-structures.json:115]:
```suggestion
```

**Comment by anshulsoni4** [comment]
done now pls check @rupali-codes

**Review by CBID2 — APPROVED** [review]
LGTM

**Comment by anshulsoni4** [comment]
can you merge the changes ?

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**
